cli/signal_unix.go

```diff
@@ -7,8 +7,24 @@ import (
 	"syscall"
 )
 
+// StopSignals are used to gracefully exit.
+// An example is exiting provisioner daemons but not canceling
+// jobs, allowing a successful and clean exit.
+var StopSignals = []os.Signal{
+	syscall.SIGTERM,
+}
+
+// InterruptSignals are used to less gracefully exit.
+// An example is canceling a job, waiting for a timeout,
+// and then exiting.
 var InterruptSignals = []os.Signal{
+	// SIGINT
 	os.Interrupt,
-	syscall.SIGTERM,
```

Comment:
This is a breaking change in all CLI commands that handle signals. They used to handle `TERM` but no longer do.